lib/active_model_serializers/support.rb

```diff
@@ -0,0 +1,12 @@
+module ActiveModelSerializers
+  module Support
+    module UrlHelpers
```

Comment:
in https://github.com/rails-api/active_model_serializers/pull/1289/files#r42808317 we were thinking this might be part of `SerializationContext`, but that seems like an extra thing to pass around.

The behavior would be something like

```ruby
class SerializationContext
  class << self
    attr_accessor :url_helpers, :default_url_options
  end
end
```

and the Railtie would

```ruby
SerializationContext.url_helpers = Rails.application.routes.url_helpers
SerializationContext.default_url_options =  ActionController::Base.default_url_options # or Rails.application.routes.default_url_options
```

So that in https://github.com/rails-api/active_model_serializers/blob/8a040052afc0430bef039cc3bc11ff900819a713/lib/active_model_serializers/adapter/json_api/link.rb you'd have

```ruby
require 'active_model_serializers/support'
require 'active_support/core_ext/module/delegation`

module ActiveModelSerializers
  module Adapter
    class JsonApi
      class Link
        include SerializationContext.url_helpers
        delegate :default_url_options, to: SerializationContext
```